send.go

```diff
@@ -50,10 +50,11 @@ func send(msg *protocol.Message, c *Channel, args interface{}) error {
 /**
 Create packet based on given data and send it
 */
-func (c *Channel) Emit(method string, args interface{}) error {
+func (c *Channel) Emit(method string, namespace string, args interface{}) error {
```

Comment:
This sends in a specific namespace? Does one have to connect to that same namespace previously?

Reply:
Yes. You need to establish the connection to that namespace before you can start emitting message in it.